src/client/scripts/createRoom.js

```diff
@@ -1,4 +1,4 @@
-/* globals Rooms:false RoomMethods:false SnackbarMethods:false Room:false*/
+/* globals Rooms:false RoomMethods:false SnackbarMethods:false Room:false DEFAULT_SNACKBAR_TIMEOUT:true*/
```

Comment:
fyi, the true modifier means that you can edit the global, we don't want that here, should probably be false